recipes/vtu11/all/conandata.yml

```diff
@@ -0,0 +1,5 @@
+sources:
+  "1.2.0":
```

Comment:
Should match the versioning used by upstream https://github.com/phmkopp/vtu11/releases/tag/v1.2

```suggestion
  "1.2":
```